tests/Behat/FeatureContext.php

```diff
@@ -216,13 +216,13 @@ public function fillInMultivalueField($column, $value, $row, $field) {
       'fifth' => '5',
       'sixth' => '6',
     ];
+    $tr = $table->find('xpath', "//tbody//tr[position()={$row_map[$row]}]");
```

Comment:
just rename of variable?

Reply:
not "just", if you check above its overwriting the value of the $row function parameter, who's actual value is needed in the error message below.